README.md

```diff
@@ -1,73 +1,243 @@
+| `#getParametersAsJSONSchema()` | Retrieve and convert the operations parameters into an array of JSON Schema schemas for each available type of parameter available on the operation: `path`, `query`, `body`, `cookie`, `formData`, and `header`. |
+| `#hasParameters()` | Determine if the operation has any parameters to send. |
+| `#hasRequiredParameters()` | Determine if any of the parameters on this operation are required. |
+<!-- prettier-ignore-end -->
+
+#### Request Body
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getRequestBody()` | Retrieve the raw request body object for a given content type. If none is specified it will return either the first that's JSON-like, or the first defined. |
+| `#getRequestBodyExamples()` | Retrieve an array of all request body examples that this operation has defined. |
+| `#getRequestBodyMediaTypes()` | Retrieve a list of all the media/content types that the operation can accept a request body payload for. |
+| `#hasRequestBody()` | Determine if this operation has a request body defined. |
+<!-- prettier-ignore-end -->
+
+#### Responses
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getResponseAsJSONSchema()` | Retrive and convert a response on this operation into JSON Schema. |
+| `#getResponseByStatusCode()` | Retrieve the raw response object for a given status code. |
+| `#getResponseExamples()` | Retrieve an array of all response examples that this operation has defined. |
+| `#getResponseStatusCodes()` | Retrieve all status codes that this operation may respond with. |
+| `#hasRequiredRequestBody()` | Determine if this operation has a required request body. |
+<!-- prettier-ignore-end -->
+
+#### Security
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getSecurity()` | Return all security requirements that are applicable for either this operation, or if the operation has none specific to it, then for the entire API. |
+| `#getSecurityWithTypes()` | Return a collection of all security schemes applicable to this operation (using `#getSecurity()`), grouped by how the security should be handled (either AND or OR auth requirements). |
+| `#prepareSecurity` | Return an object of every security scheme that _can_ be used on this operation, indexed by the type of security scheme it is (eg. `Basic`, `OAuth2`, `APIKey`, etc.). |
+<!-- prettier-ignore-end -->
+
+#### Specification Extensions
+
+<!-- prettier-ignore-start -->
+| Method | Description |
+| :--- | :--- |
+| `#getExtension()` | Retrieve a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions) if it exists on this operation. |
+| `#hasExtension()` | Determine if a given [specification extension](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#specificationExtensions) exists on this operation. |
+<!-- prettier-ignore-end -->
+
+### Callbacks
```

Comment:
Could this section be consolidated with [this one](https://github.com/readmeio/oas/tree/feat/sunset-cli#callbacks)? The hierarchy feels a bit confusing with the placement of this section since callbacks are technically a subset of operations.

Reply:
I don't think I can do this because callbacks in OAS can actually have their own callbacks so merging the `getIdentifier()` docs into this one I don't think fits because it's only available on the `Callback` instance.

Reply:
callbacks: not even once